cmake/onnxruntime.cmake

```diff
@@ -26,16 +26,29 @@ add_custom_command(OUTPUT ${SYMBOL_FILE} ${CMAKE_CURRENT_BINARY_DIR}/generated_s
 
 add_custom_target(onnxruntime_generate_def ALL DEPENDS ${SYMBOL_FILE} ${CMAKE_CURRENT_BINARY_DIR}/generated_source.c)
 if(WIN32)
-    add_library(onnxruntime SHARED ${SYMBOL_FILE} "${ONNXRUNTIME_ROOT}/core/dll/dllmain.cc")
+    add_library(onnxruntime SHARED 
+      ${SYMBOL_FILE}
+      "${ONNXRUNTIME_ROOT}/core/dll/dllmain.cc"
+      ${PROJECT_SOURCE_DIR}/onnxruntime.rc
+    )
 else()
-    add_library(onnxruntime SHARED ${CMAKE_CURRENT_BINARY_DIR}/generated_source.c)
+    add_library(onnxruntime SHARED 
+      ${CMAKE_CURRENT_BINARY_DIR}/generated_source.c
+      ${PROJECT_SOURCE_DIR}/onnxruntime.rc
```

Comment:
Are resources supported outside of WIN32?

Reply:
they arn't i dont believe there are even apis for accessing them on other platforms.

---
In reply to: [375562814](https://github.com/microsoft/onnxruntime/pull/2978#discussion_r375562814) [](ancestors = 375562814)

Reply:
ah I though WIN32 was filtering x86, but now I see it is filtering any Windows build. I'll fix that.